submissions/DaniaB24/html-css-popup/index.html

```diff
@@ -0,0 +1,115 @@
+                        </li>
+                        <li class="li_popup" tabindex="11">
+                            <p class="p_popup"><img src="/images/apps.jpg" class="popup">Apps</p>
+                        </li>
+                        <li class="li_popup" tabindex="12">
+                            <p class="p_popup"><img src="/images/documents.png" class="popup">Documents</p>
+                        </li>
+                        <li class="hr_line"></li>
+                        <li class="li_popup" tabindex="13">
+                            <p class="p_popup"><img src="/images/chat.png" class="popup" id="popup_chat">Chat</p>
+                        </li>
+                        <li class="li_popup" tabindex="14">
+                            <p class="p_popup"><img src="/images/classroom.png" class="popup" id="popup_classroom">Classroom</p>
+                        </li>
+                        <li class="li_popup" tabindex="15">
+                            <p class="p_popup"><img src="/images/collection.png" class="popup" id="popup_collection">Collection</p>
+                        </li>
+                        <li class="li_popup" tabindex="16">
+                            <p class="p_popup"><img src="/images/mars.png" class="popup">Mars</p>
+                        </li>
+                        <li class="li_popup" tabindex="17">
+                            <p class="p_popup"><img src="/images/blogs.png" class="popup">Blogs</p>
+                        </li>
+                        <li class="li_popup" tabindex="18">
+                            <p class="p_popup"><img src="/images/ads.png" class="popup">Ads</p>
+                        </li>
+                        <li><a class="kottans_link" href="https://kottans.org/" target="_blank">Even more from Kottans</a></li>
+                    </ul>
+                </div>
+
+            </li>
+            <li class="nav__item"><img src="/images/bell.png" class="bell"></li>
+            <li class="nav__item"><img src="/images/ava.jpg" class="avatar"></li>
+
+        </ul>
+    </nav>
+    <!--
+  <input class="check__input2" id="more-btn" type="checkbox">
+  <label for="more-btn" class="extra__label">More</label>
+  <div class="menu_2">
+    <ul class="menu-bottom">
+      <li class="li_popup">
+        <p><img src="/images/chat.png" class="popup">Chat</p>
+      </li>
+      <li class="li_popup">
+        <p><img src="/images/classroom.png" class="popup">Classroom</p>
+      </li>
+      <li class="li_popup">
+        <p><img src="/images/collection.png" class="popup">Collection</p>
+      </li>
+      <li class="li_popup">
+        <p><img src="/images/mars.png" class="popup">Mars</p>
+      </li>
+      <li class="li_popup">
+        <p><img src="/images/blogs.png" class="popup">Classroom</p>
+      </li>
+      <li class="li_popup">
+        <p><img src="/images/ads.png" class="popup">Ads</p>
+      </li>
+    </ul>
+  </div>
+  -->
+</body>
+
+</html>
```

Comment:
Could you please add an empty line at the end of the file? When you see this symbol it means the empty line is needed   
![empty line](https://user-images.githubusercontent.com/10871447/100141646-6b4dc700-2e9b-11eb-8258-361247ad8d04.png)
https://github.com/kottans/frontend-2021-homeworks/pull/165/files